lib/raven/integrations/rack.rb

```diff
@@ -82,6 +84,11 @@ def from_rack(env_hash)
 
     private
 
+    # Request ID based on ActionDispatch::RequestId
+    def read_request_id_from(env_hash)
+      env_hash['action_dispatch.request_id'] || env_hash['HTTP_X_REQUEST_ID']
```

Comment:
We should make sure the keys we use here syncs with the `REQUEST_ID_HEADERS` constant automatically. So I prefer something like

```ruby
def read_request_id_from(env_hash)
  REQUEST_ID_HEADERS.each do |key|
    request_id = env_hash[key]
    return request_id if request_id
  end
  nil
end
```

Reply:
single source of truth makes sense~ : )